client/app/home/menu/menu.html

```diff
@@ -0,0 +1,9 @@
+<button ng-click="create()" ng-show="isState('home.menu')">Create new board</button>
+<div ui-view></div>
+
+<div class="container">
+  <div ui-sref="home.board({id: board.id})" class="board" ng-repeat="board in boards | orderBy:'-createdAt'">
```

Comment:
On click:
1. change state to `board`
2. pass in a parameter `id` that is equal to `board.id`